### PR TITLE
Twilio Event Streams: Sink and subscription via TG

### PR DESCRIPTION
**Pull request by janorivera**
## Description

This PR pairs with[ this one](https://github.com/techmatters/infrastructure-config/pull/379).
This is PR is based on this [Twilio Event Example](https://github.com/twilio/terraform-provider-twilio/blob/main/examples/events/v1/events.tf).

The idea is to manage and create Sinks and Subscriptions for [Twilio Event Streams ](https://www.twilio.com/docs/events/webhook-quickstart) via Terragrunt.

- Default event subscriptions will be handled in the defaults.hcl file. This can be overwritten in each helpline file if needed.
- The structure allows for creating multiple subscriptions

### Checklist
- [ ] Corresponding issue has been opened
- [ ] New tests added
- [ ] Feature flags added
- [ ] Strings are localized
- [ ] Tested for chat contacts
- [ ] Tested for call contacts

### Other Related Issues

None

### AFTER YOU MERGE

1. Cut a release tag using the Github workflow. Wait for it to complete and notify in the #aselo-deploys Slack channel.
2. Comment on the ticket with the release tag version AND any additional instructions required to configure an environment to test the changes.
3. Only then move the ticket into the QA column in JIRA

You are responsible for ensuring the above steps are completed. If you move a ticket into QA without advising what version to test, the QA team will assume the latest tag has the changes. If it does not, the following confusion is on you! :-P

### CODE REVIEW

File: twilio-iac/terraform-modules/events/v1/variables.tf [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  type        = string
+}
+
+variable "short_helpline" {
+  description = "Short (usually 2 letter) upper case code for helpline"
+  type        = string
+}
+
+variable "default_webhook_url" {
+  description = "The human readable helpline name."
```
Comment (stephenhand) [inline twilio-iac/terraform-modules/events/v1/variables.tf:12]: Is that description correct?

File: twilio-iac/terraform-modules/events/v1/main.tf [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+resource "twilio_events_subscriptions_v1" "subscription" {
+  for_each    = var.subscriptions
+  description = "${title(replace(each.key, "_", " "))} ${upper(var.short_helpline)}_${upper(var.short_environment)} Events Subscription"
+  sink_sid    = twilio_events_sinks_v1.webhook_sink[each.key].sid
+  types       = [for event in each.value.events : jsonencode({ type = event.type })]
+}
+/*
+I'm leaving this out as this resource doesn't work well.
```
Comment (stephenhand) [inline twilio-iac/terraform-modules/events/v1/main.tf:31]: Does this need to be a manual step for now then? Perhaps raise an issue of the twilio terraform provider repo if you haven't already, just for visibility if nothing else

**Review by stephenhand — APPROVED** [review]

File: twilio-iac/terraform-modules/events/v1/variables.tf (hunk 1 above)
Comment (janorivera) [reply twilio-iac/terraform-modules/events/v1/variables.tf]: nope!

File: twilio-iac/terraform-modules/events/v1/main.tf (hunk 2 above)
Comment (janorivera) [reply twilio-iac/terraform-modules/events/v1/main.tf]: No, so the only difference now is that to edit/add events to subscription we'll need to recreate it.

**Pull request merged**